### ISSUE TITLE
Documentation for Custom Cache Implementations

### ISSUE THREAD

**Issue by fbartho** [apollographql/apollo-feature-requests#47]
Migrated from: apollographql/apollo-client#2484

Currently, the InMemoryCache is a great start, but people often need more control. A specific feature that is desired is the ability to have a defined serialization format, so that things like apollo-cache-persist could strip certain "sensitive" data before serializing to disk. Other examples of data that may not be wanted in a disk cache: "App Navigation State"

It would be great if we could have documentation for custom cache implementations to rely on, so variants of the InMemoryCache can be used.

**Comment by qswitcher** [comment]
I recently documented this in a blog post https://medium.com/@jeffreyrussom/creating-your-own-apollo-cache-for-perf-and-profit-8f786849e5f6 if this is what you were looking for.

**Comment by fbartho** [comment]
Yeah! That’s a great start @qswitcher ! If you could contribute that as official docs, I think people would love to add on from there!
